docs/mayastor-concept.md

```diff
@@ -0,0 +1,170 @@
+
+Further detail regarding the implementation of CSI driver components and their function can be found within the Kubernetes CSI Developer Documentation.
+
+#### NATS
+
+NATS is a high performance open source messaging system.  It is used within Mayastor as the transport mechanism for registration messages passed between Mayastor I/O engine pods running in the cluster and the MOAC component which maintains an inventory of active Mayastor nodes and reflects this via CRUD actions on MayastorNode custom resources.
+
+In future releases of Mayastor, the control plane will transition towards a more microservice-like architecture following the saga pattern, whereupon a highly available NATS deployment within the Mayastor namespace will be employed as an event bus.
+<br><br>
+----
+## Scope
+
+This quickstart guide describes the actions necessary to perform a basic installation of Mayastor on an existing Kubernetes cluster, sufficient for evaluation purposes.  It assumes that the target cluster will pull the Mayastor container images directly from MayaData public container repositories.  Where preferred, it is also possible to [build Mayastor locally from source](https://github.com/openebs/Mayastor/blob/develop/doc/build.md) and deploy the resultant images but this is outside of the scope of this quickstart document.
+
+Deploying and operating Mayastor within production contexts requires a foundational knowledge of Mayastor internals and best practices, found elsewhere within this documentation resource.  Some application use cases may require specific configuration and where this is so, it is called out in the Use Cases section.
+
+
+>**Mayastor is beta software**.  It is considered largely, if not entirely, feature complete.  Beta software "will generally have many more bugs in it than completed software and speed or performance issues, and may still cause crashes or data loss"
+----
+
+## Known Issues
+
+### Live Issue Tracker
+
+Mayastor is currently considered to be beta software.
+
+> "(it) will generally have many more bugs in it than completed software and speed or performance issues, and may still cause crashes or data loss."
+
+The project's maintainers operate a live issue tracking dashboard for defects which they have under active triage and investigation.  It can be accessed [here](https://mayadata.atlassian.net/secure/Dashboard.jspa?selectPageId=10015).  You are strongly encouraged to familiarise yourself with the issues identified there before using Mayastor and when raising issue reports in order to limit to the extent possible redundant issue reporting. 
```

Comment:
```suggestion
The project's maintainers operate a live issue tracking dashboard for defects that they have under active triage and investigation.  It can be accessed [here](https://mayadata.atlassian.net/secure/Dashboard.jspa?selectPageId=10015).  You are strongly encouraged to familiarize yourself with the issues identified there before using Mayastor and when raising issue reports in order to limit to the extent possible redundant issue reporting. 
```